bin/image_dart_compiler.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:async';
+import 'dart:io';
+import 'package:scissors/src/image_inlining/image_dart_compiling.dart';
+
+Future<ImageInformation> fromFile(String fileName) async {
+  return new ImageInformation(fileName, await new File(fileName).readAsBytes());
+}
+
+/// Command-line entry point. Takes list of paths to pictures, producing Dart
+/// source on stdout.
+main(List<String> arguments) async {
+  try {
+    String source =
+        await generateDartSource(await Future.wait(arguments.map(fromFile)));
+    stdout.write(source);
+  } on ConversionException catch (e) {
+    stderr.write("An error occurred while converting images to Dart.\n");
```

Comment:
The idea is that we know that this particular exception is thrown by callee and this is not an unexpected situation. Other kinds of exceptions might easily be programming errors and I'd prefer to see default error message with a stacktrace for these.